pkg/apis/sources/v1alpha1/containersource_types.go

```diff
@@ -117,6 +117,16 @@ func (s *ContainerSourceStatus) MarkNoSink(reason, messageFormat string, message
 	containerCondSet.Manage(s).MarkFalse(ContainerConditionSinkProvided, reason, messageFormat, messageA...)
 }
 
+// IsDeployed returns true if the Deployed condition has status true, otherwise
+// false.
+func (s *ContainerSourceStatus) IsDeployed() bool {
+	c := containerCondSet.Manage(s).GetCondition(ContainerConditionDeployed)
+	if c == nil || !c.IsTrue() {
```

Comment:
I think the negation of `IsTrue()` rubs me the wrong way.

How about? 

```golang
if c != nil {
  return c.IsTrue();
}
return false
```

Reply:
I copied that from [the `IsHappy` method from knative/pkg](https://github.com/grantr/eventing-sources/blob/04b2f641a1e255b736cd4abae00553c12ac3b49f/vendor/github.com/knative/pkg/apis/duck/v1alpha1/condition_set.go#L164-L169). Not sure why it's written that way there, but I agree yours is better.